twilight-model/src/application/mod.rs

```diff
@@ -1,3 +1,4 @@
 pub mod command;
+pub mod emoji;
```

Comment:
I don't see Discord adding enough endpoints to warrant us grouping stuff into a separate `emoji` module (and adding one is backwards compatible, should that change).